packages/loader/container-loader/src/container.ts

```diff
@@ -555,6 +568,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         this.clientDetailsOverride = config.clientDetailsOverride;
         this._id = config.id;
         this._resolvedUrl = config.resolvedUrl;
+        this.maxClientLeaveWaitTime = config.maxClientLeaveWaitTime;
```

Comment:
seem weird that we are expanding all the config properties where, just make config a private readonly member as use the value off the config. we should also clean up the other expansions, but that can be done in a follow up

Reply:
Removed.